packages/web-components/@sfx/base/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- SFX-152: Added the Base class.
+  - It adds slot functionality.
```

Comment:
Perhaps you could explain more generally what this class does or is intended for. "The `Base class is intended to be used as a parent class for all SF-X components. It includes features such as [...]"